stdlib/Profile/src/Allocs.jl

```diff
@@ -42,7 +42,7 @@ macro profile(opts, ex)
     _prof_expr(ex, opts)
 end
 macro profile(ex)
-    _prof_expr(ex, :(skip_every=1000))
+    _prof_expr(ex, :(sample_rate=1.0))
```

Comment:
```suggestion
    _prof_expr(ex, :(sample_rate=0.0001))
```
I think we should still keep a small default sample rate, just like the default large skip every was before, yeah?
(1/1000 = 0.0001)